integration-cli/docker_cli_commit_test.go

```diff
@@ -37,6 +38,21 @@ func (s *DockerSuite) TestCommitWithoutPause(c *check.C) {
 	dockerCmd(c, "inspect", cleanedImageID)
 }
 
+func (s *DockerSuite) TestCommitWithoutCgroupFreeze(c *check.C) {
+	testRequires(c, DaemonIsLinux)
+	outId, _ := dockerCmd(c, "run", "-i", "-d", "busybox")
+
+	containerID := strings.TrimSpace(outId)
+	output, _ := dockerCmd(c, "exec", containerID, "cat /proc/self/cgroup | grep freeze | awk -F : '{print $3}'")
+
+	freezePath := strings.TrimSpace(output)
+	os.RemoveAll(strings.Join([]string{"/sys/fs/cgroup/freeze", freezePath}, ""))
```

Comment:
Perhaps the path should be `/sys/fs/cgroup/freezer`?
And a cgroup hierarchy can't be deleted via `rm -rf`, instead we should first move the tasks in the cgroup to its parent hierarchy and then remove the directory via `rmdir`.
FYI http://blog.tinola.com/?e=21

It's weird why the jenkins result isn't showing in the status of this PR.

I think the `dockerCmd` below won't touch the changes that this PR introduced.  It should be `docker commit --pause freezetest` and the command should fail, right?